code/modules/jobs/job_types/wasteland.dm

```diff
@@ -91,6 +91,61 @@
 	ADD_TRAIT(H, TRAIT_HARD_YARDS, src)
 	ADD_TRAIT(H, TRAIT_PA_WEAR, src)
 
+/datum/job/wasteland/enclavesci
+	title = "Enclave Scientist"
+	flag = F13USSCIENTIST
+	faction = "Enclave"
+	total_positions = 1
+	spawn_positions = 1
+	description = "You're responsible for the maintenance of the base, the knowledge you've accumulated over the years is the only thing keeping the remnants alive. You've dabbled in enough to be considered a Professor in proficiency, but they call you Doctor. Support your dwindling forces and listen to the Lieutenant."
+	forbids = ""
```

Comment:
Should probably make it abundantly clear that they cannot betray the Enclave, because someone will do it

Reply:
deserting/betraying without a good reason is already something people get banned for, we shouldn't need to put it on every role. it's simple to understand that the faction with 3(4 if this gets merged) slots is something you're supposed to be tightknit with.